dev-libs/jemalloc/files/jemalloc-5.3.0-backport-pr-2338.patch

```diff
@@ -0,0 +1,78 @@
+From aba1645f2d65a3b5c46958d7642b46ab3c142cf3 Mon Sep 17 00:00:00 2001
```

Comment:
Please include a link at the top of the patch to the relevant upstream commit(s) + PR.